steamos-waydroid-installer.sh

```diff
@@ -291,7 +294,22 @@ else
 		elif [ "$Choice" == "TV" ]
 		then
 			echo Android TV chosen!
-			exit
+			echo Initializing Waydroid
+			echo -e "$current_password\n" | sudo -S waydroid init
+			echo Downloading Android TV image
+            echo -e "$current_password\n" | sudo -S curl -o ~/waydroid/images/androidtv.zip $ANDROID_TV_IMG -L
+			hash=$(md5sum "/home/deck/waydroid/images/androidtv.zip" | awk '{print $1}')
+			# Verify the MD5 hash
+			if [[ "$hash" != "$ANDROID_TV_IMG_MD5" ]]; then
+  				echo MD5 hash mismatch for Android TV image, indicating a corrupted download. This might be due to a network error, you can try again.
+  				cleanup_exit
+			fi
+
+			echo Extracting Archive
+			echo -e "$current_password\n" | sudo -S unzip -o ~/waydroid/images/androidtv -d ~/waydroid/images
+			echo -e "$current_password\n" | sudo -S rm ~/waydroid/images/androidtv.zip
+			echo Reinitializing Waydroid
+			echo -e "$current_password\n" | sudo -S waydroid init -f
```

Comment:
```suggestion
			echo Initializing Waydroid
			echo -e "$current_password\n" | sudo -S waydroid init
```

this can just be a regular init now per above